ui/src/store/program/slices/programDataSlice.ts

```diff
@@ -37,17 +37,14 @@ export const createProgramDataSlice: ProgramStateCreator<ProgramDataSlice> = ()
 
 /** Pokud ještě není dotažený tak dotáhne rok, příhlášen se dotahuje vždy */
 export const načtiRok = async (rok: number) => {
-  const [aktivityPřihlášen, aktivity] =
+  const [aktivity] =
     await Promise.all([
-      fetchAktivityPřihlášen(rok),
       fetchAktivity(rok)
     ]);
 
   useProgramStore.setState(s => {
     for (const aktivita of aktivity) {
       s.data.aktivityPodleId[aktivita.id] = { ...s.data.aktivityPodleId[aktivita.id], ...aktivita, [DOTAŽENO]: Date.now() };
-    }
-    for (const aktivita of aktivityPřihlášen) {
```

Comment:
tady se jedná o jinou kolekci, (`aktivity` vs `aktivityPřihlášen`) takže to tu má zůstad přes 2x foreach